logpolicy/logpolicy.go

```diff
@@ -70,8 +70,18 @@ func getLogTarget() string {
 	return getLogTargetOnce.v
 }
 
+// LogURL is the base URL for the configured logtail server, or the default.
```

Comment:
Document & guarantee that it doesn't end in a slash so callers can blindly append "/foo" and not worry about double slashes.

Reply:
Done.